src/components/Bar.jsx

```diff
@@ -11,10 +11,11 @@ import { shouldEnableTracking, getTracker, configureTracker } from 'cozy-ui/reac
 import Claudy from './Claudy'
 
 class Bar extends Component {
-  constructor (props) {
+  constructor (props, context) {
     super(props)
+    this.store = context.store
     this.state = {
-      claudyConfig: null, // no claudy by default
+      claudyActions: null, // no claudy by defaul
```

Comment:
s/defaul/default/

(in any case I guess `claudyActions: null` has got enough meaning and the comment is useless)